pages/lists/[listId]/contributors.tsx

```diff
@@ -78,13 +68,17 @@ const ContributorsListPage = ({ list, initialData, isError, isOwner }: Contribut
       {isError ? (
         <Error errorMessage="Unable to load list of contributors" />
       ) : (
-        <ContributorsList
-          contributors={contributors}
-          meta={meta}
-          isLoading={isLoading}
-          setPage={setPage}
-          range={String(range) ?? "30"}
-        />
+        <ErrorBoundary
+          fallback={<div className="grid place-content-center">Error loading the list of contributors</div>}
```

Comment:
I went with a barebones fallback error message @isabensusan, but going forward a bit of styling for this would be nice as we can use this for graphs or other data driven components.